includes/html/pages/device/port.inc.php

```diff
@@ -81,11 +85,13 @@
     $menu_options['sensors'] = 'Health';
 }
 
-if (dbFetchCell("SELECT COUNT(*) FROM `ports_adsl` WHERE `port_id` = '" . $port->port_id . "'")) {
-    $menu_options['adsl'] = 'ADSL';
+if (PortAdsl::where('port_id', $port->port_id)->count()) {
```

Comment:
You can use ->exists() here instead of ->count() because we don't care about the count, just if it is more than 0